src/server/scripts/Commands/cs_misc.cpp

```diff
@@ -2449,6 +2454,7 @@ class misc_commandscript : public CommandScript
 
     static bool HandleFreezeCommand(ChatHandler* handler, char const* args)
     {
+        Unit* target = handler->getSelectedUnit();
```

Comment:
check if null check is required

Reply:
got a check in line 2534, hopefully it is what you ment. If not then im a bit confused in how you would want the null check to be done exactly in this case.